xoa/resource_xenorchestra_vm.go

```diff
@@ -244,6 +244,13 @@ $ xo-cli xo.getAllObjects filter='json:{"id": "cf7b5d7d-3cd5-6b7c-5025-5c935c8cd
 			Description: "Whether terraform should wait until IP addresses are present on the VM's network interfaces before considering it created. This only works if guest-tools are installed in the VM. Defaults to false.",
 			Optional:    true,
 		},
+		"bootaftercreate": &schema.Schema{
+			Type:             schema.TypeBool,
+			Default:          true,
+			Description:      "Whether Terraform should boot the VM after creation. Defaults to true.",
```

Comment:
We are currently partially migrated to tfplugindocs (see #256 for details), but that means that the `Description` string here should match what is in the `docs/resources/vm.md` file. That will make the eventual migration easier.